Gemfile

```diff
@@ -0,0 +1,6 @@
+source "https://rubygems.org"
+
+group :test do
+  gem 'rspec'
```

Comment:
Agree, but is a general good practice to block at less on big branches.... keep in mind semantic versioning, 3.x can break api with 2.x, but ei! just me trying to enforce this, nothing else ...

otherwise keep in mind what happened to us with jar-dependencies or the aws-sdk, one problem there on our side was not to block to a given version.